routes/index.js

```diff
@@ -134,7 +139,8 @@ router.post('/captions/start', async function (req, res) {
         'Content-Type': 'application/json',
       },
     });
-    res.send(captionResponse.data.captionsId);
+    captionsId = captionResponse.data.captionsId;
```

Comment:
This is not a blocker for the PR. Going forward we are looking to use `node-fetch` instead of `axios`. This way it is more in sync with the browser's fetch API